src/index.html

```diff
@@ -7,8 +7,70 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav-header">
+      <div class="nav-wrapper">
+        <div class="nav-logo">
+          <img src="images/logo.png" alt="logo">
+        </div>
+        <nav class="nav">
+          <ul class="nav-list">
+            <li class="nav-item">
+              <a
+                href="#"
+                class="nav-link is-active">
+                Apple
+              </a>
```

Comment:
if you have only 2 attributes you don't need move tham to a new line
```suggestion
              <a href="#" class="nav-link is-active">
                Apple
              </a>
```